packages/x-components/src/components/suggestions/__tests__/base-suggestion.spec.ts

```diff
@@ -14,18 +14,27 @@ import { createPopularSearch } from '../../../__stubs__/popular-searches-stubs.f
 function renderBaseSuggestion({
   query = 'bebe',
   suggestion = createPopularSearch('bebe lloron'),
-  suggestionSelectedEvents = {}
+  suggestionSelectedEvents = {},
+  template = '<BaseSuggestion :query="query" :suggestion="suggestion" ' +
```

Comment:
We're always using the same template across all tests, so you can set it inside the function